_entries/02-03 mongodb.md

```diff
@@ -21,10 +21,10 @@ There are two templates available:
 > oc get templates -n openshift
 > ```
 
-Create a mongoDB deployment using the `mongodb-persistent` template. You're passing in the values to be replaced (username, password and database) which generates a YAML/JSON file. You then pipe it to the `oc create` command.
+Create a mongoDB deployment using the `mongodb-ephemeral` template. You're passing in the values to be replaced (username, password and database) which generates a YAML/JSON file. You then pipe it to the `oc create` command.
```

Comment:
Curious, why ephemeral?

Reply:
see first description of the PR. I don't why, but I was trying on the MS playground and PV was readonly while it should not. I tried to play with uit without success. Since PV is covered in details in the following chapter, I thought that it would not change the value of the step using ephemeral, because the difference of the two modes is explained two lines before. 

I have now my own cluster, I will retry to see, if it works and then revert.

Reply:
cannot try again, no pull secret on my cluster :/
I revert then